modules/ingest-common/src/main/java/org/elasticsearch/ingest/common/ScriptProcessor.java

```diff
@@ -95,8 +96,7 @@ public Factory(ScriptService scriptService) {
         public ScriptProcessor create(Map<String, Processor.Factory> registry, String processorTag,
                                       Map<String, Object> config) throws Exception {
             XContentBuilder builder = XContentBuilder.builder(JsonXContent.jsonXContent).map(config);
-            JsonXContentParser parser = new JsonXContentParser(NamedXContentRegistry.EMPTY,
-                JSON_FACTORY.createParser(builder.bytes().streamInput()));
+            XContentParser parser = JsonXContent.jsonXContent.createParser(NamedXContentRegistry.EMPTY, builder.bytes().streamInput());
```

Comment:
Why does this one use `JsonXContent.jsonXContent.createParser(...)` and the one below use `XContentType.JSON.xContent().createParser(...)` ? They're the same thing so I think we should standardize on one (probably the XContentType one?)

Reply:
I suppose because I'm used to the top one and forgot about the bottom one. I'll make them line up.